src/test/kotlin/rys/ajaxpetproject/controller/MongoUserControllerTest.kt

```diff
@@ -0,0 +1,106 @@
+package rys.ajaxpetproject.controller
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import lombok.AllArgsConstructor
+import lombok.NoArgsConstructor
+import lombok.RequiredArgsConstructor
+import org.bson.types.ObjectId
+import org.hamcrest.Matchers.anyOf
+import org.hamcrest.Matchers.`is`
+import org.junit.jupiter.api.Test
+import org.mockito.Mockito.`when`
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest
+import org.springframework.boot.test.mock.mockito.MockBean
+import org.springframework.http.MediaType
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder
+import org.springframework.test.web.servlet.MockMvc
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers.status
+import rys.ajaxpetproject.model.MongoUser
+import rys.ajaxpetproject.service.UserService
+import java.util.*
+
+@WebMvcTest(UserController::class)
+@RequiredArgsConstructor
+@AllArgsConstructor
+@NoArgsConstructor
+class MongoUserControllerTest {
+    @MockBean
+    lateinit var userService: UserService
+
+    @Autowired
+    lateinit var mockMvc: MockMvc
+
+    @Test
+    fun getUserSuccessfullyTest() {
+        val testMongoUser = MongoUser( id = ObjectId(Date(1694093823L * 1000)),
+            userName = "testUser", password =
+            BCryptPasswordEncoder().encode("testPassword")
+        )
```

Comment:
```suggestion
        val testMongoUser = MongoUser(
            id = ObjectId(Date(1694093823L * 1000)),
            userName = "testUser",
            password = BCryptPasswordEncoder().encode("testPassword")
        )
```

it's better to have such formatting where every parameter is on the separate line

Reply:
Done